cmd/src/campaigns_preview.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/sourcegraph/src-cli/internal/campaigns"
+	"github.com/sourcegraph/src-cli/internal/output"
+)
+
+func init() {
+	usage := `
+'src campaigns preview' is used to create a campaign spec on a Sourcegraph
```

Comment:
I think this one here, combined with the description for `src campaigns apply` makes things clearer:

```suggestion
'src campaigns preview' is executes the steps in a campaign spec and uploads it to a Sourcegraph
```